src/content/roadmap/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Ethereum roadmap
+description: The path to more scalability, security and sustainability for Ethereum.
+lang: en
+template: roadmap
+image: ../../assets/upgrades/merge.png
+alt: "Ethereum roadmap"
+summaryPoints:
+buttons:
+  - label: Further updates
+    toId: what-changes-are-coming
+  - label: Past updates
+    to: /history/
+    variant: outline
+---
+
+Ethereum is already a powerful platform for global coordination, but it is still being improved. An ambitious set of improvements will upgrade Ethereum from its current form into a fully scaled, maximally resilient platform. These upgrades are laid out in the Ethereum roadmap.
+
+**To learn about previous upgrades to Ethereum, please visit our [History](/history/) page**
+
+## What changes are coming to Ethereum? {#what-changes-are-coming}
+
+The Ethereum roadmap outlines the specific improvements that will be made to protocol in the future. Overall, the roadmap will bring the following benefits to Ethereum users:
+
+<CardGrid>
+  <RoadmapActionCard
+    to="/roadmap/scaling"
+    title="Cheaper transactions"
+    image="scaling"
+    description="Rollups are too expensive and rely on centralized components, causing users to place too much trust in their operators. The roadmap includes fixes for both of these problems."
+  >
+    <ButtonLink to="/roadmap/scaling/">Read more</ButtonLink>
+  </RoadmapActionCard>
+  <RoadmapActionCard
+    to="/roadmap/security"
+    title="Extra security"
+    image="security"
+    description="Ethereum is already very secure but it can be made even stronger, ready to withstand all kinds of attack far into the future."
+  >
+    <ButtonLink to="/roadmap/security/">Read more</ButtonLink>
+  </RoadmapActionCard>
+  <RoadmapActionCard
+    to="/roadmap/user-experience"
+    title="Better user experience"
+    image="userExperience"
+    description="More support for smart contract wallets and light-weight nodes will make using Ethereum simpler and safer."
+  >
+    <ButtonLink to="/roadmap/user-experience/">Read more</ButtonLink>
+  </RoadmapActionCard>
+  <RoadmapActionCard
+    to="/roadmap/future-proofing"
+    title="Future proofing"
+    image="futureProofing"
+    description="Ethereum researchers and developers are solving tomorrow's problems today, readying the network for future generations."
+  >
+    <ButtonLink to="/roadmap/future-proofing/">Read more</ButtonLink>
```

Comment:
```suggestion
    <ButtonLink to="/roadmap/scaling/">More on reducing fees</ButtonLink>
  </RoadmapActionCard>
  <RoadmapActionCard
    to="/roadmap/security"
    title="Extra security"
    image="security"
    description="Ethereum is already very secure but it can be made even stronger, ready to withstand all kinds of attack far into the future."
  >
    <ButtonLink to="/roadmap/security/">More on security</ButtonLink>
  </RoadmapActionCard>
  <RoadmapActionCard
    to="/roadmap/user-experience"
    title="Better user experience"
    image="userExperience"
    description="More support for smart contract wallets and light-weight nodes will make using Ethereum simpler and safer."
  >
    <ButtonLink to="/roadmap/user-experience/">More on user experience</ButtonLink>
  </RoadmapActionCard>
  <RoadmapActionCard
    to="/roadmap/future-proofing"
    title="Future proofing"
    image="futureProofing"
    description="Ethereum researchers and developers are solving tomorrow's problems today, readying the network for future generations."
  >
    <ButtonLink to="/roadmap/future-proofing/">More on future proofing</ButtonLink>
```